README.md

```diff
@@ -433,6 +433,38 @@ dist(0);
 // ! TypeError: ‘dist’ expected a value of type { x :: FiniteNumber, y :: FiniteNumber } as its first argument; received 0
 ```
 
+#### `Nullable`
+
+`Nullable` is used to construct types which include `null` as a member.
+
+To define `t :: Nullable a` one must provide:
+
+  - the type of `a` (exposed as `t.$1`).
+
+```haskell
+Nullable :: Type -> Type
+```
+
+For example:
+
+```javascript
+//    toUpper :: Nullable String -> Nullable String
+const toUpper =
+def('toUpper',
+    {},
+    [$.Nullable($.String), $.Nullable($.String)],
+    R.unless(R.equals(null), R.toUpper));
```

Comment:
Do we want the docs for `sanctuary-def` to assume a deep familiarity with Ramda? I use it every day and I still had to look up what `R.unless` does just to be sure I was reading the example properly.